codegen/collector/compiler.go

```diff
@@ -146,6 +146,7 @@ func (p *protoCompiler) addDescriptorsForFile(addDescriptor func(f DescriptorWit
 
 var defaultGogoArgs = []string{
 	"plugins=grpc",
+	"Mgithub.com/solo-io/solo-kit/api/external/envoy/api/v2/discovery.proto=github.com/envoyproxy/go-control-plane/envoy/api/v2",
```

Comment:
One has the discovery.proto file name and the other doesn't. Is this intentional?

Reply:
I believe so, that's what it's imported as, eg: https://github.com/solo-io/solo-apis/blob/81a712d752f0f38e95b529a864460f8199a2614b/pkg/api/gloo.solo.io/v1/enterprise/ratelimit.pb.go#L17